index.html

```diff
@@ -200,18 +200,20 @@ <h1 class="skills-header"><span><img src="./images/works/Rectangle-three.svg" al
             </ul>
     </div>
 </section>
+
 <!-- Get in touch section -->
+
 <section>
 <div class="contact-section">
     <div class="message-box">
         <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
     </div>
     <div class="form-container">
-        <form action="POST">
-            <input type="text" placeholder="Full name">
-            <input type="email" placeholder="Email">
-            <textarea name="contact" id="text-msg" cols="30" rows="5" placeholder="Enter text here"></textarea>
-            <input type="button" value="Get in touch" id="btn-submit">
+        <form action="https://formspree.io/f/mknkennj">
```

Comment:
- [ ] Your form is still not submitting because method value should equals POST
![Screenshot from 2021-09-29 01-35-54](https://user-images.githubusercontent.com/62705947/135179772-2c853df8-2b4e-44d7-8ef8-1eb1419b386e.png)

Reply:
@GzimAsani and @carlos-ssh,
can u please check if changes have been made successfully?
thanks for your feedback.